user-guide/Advanced_Modules/Data_Aggregator/Data_Aggregator_settings.md

```diff
@@ -40,6 +40,24 @@ For example:
 > - Keep in mind that **if there is no firewall in place, anyone can use the web UI and the REST API**, as no authentication is required to use Data Aggregator.
 > - To go to the debug page, add `/debug` to the URL. For example, if the URL was defined as `http://127.0.0.1:5000`, you can access the debug page at `http://127.0.0.1:5000/debug`.
 
+## GQI DxM
+
+By default, queries are executed using CoreGateway and SLHelper.
+To execute the queries using the [GQI DxM](xref:GQI_DxM), the following setting must be enabled:
+
+```json
+{
+  "QueryExecutorOptions": {
+    "UseGQIDxM": true
+  }
+}
+```
+
+> [!NOTE]
+>
+> - This setting requires Data Aggregator version 3.1.0 or higher.
```

Comment:
Maybe we should phrase it more "positively"?

```suggestion
> - This setting is available from Data Aggregator version 3.1.0 onwards.
```

Reply:
Done